sdks/java/extensions/avro/src/main/java/org/apache/beam/sdk/extensions/avro/schemas/utils/AvroUtils.java

```diff
@@ -486,10 +518,17 @@ public static Schema toBeamSchema(org.apache.avro.Schema schema) {
     return builder.build();
   }
 
+  @EnsuresNonNullIf(
+      expression = {"#1"},
+      result = false)
+  private static boolean isNullOrEmpty(@Nullable String str) {
+    return str == null || str.isEmpty();
+  }
```

Comment:
what is wrong with Strings.isNullOrEmpty?

Reply:
for some reason the checkerframework doesn't understand the nullness assertion when using it:
```
/usr/local/google/home/mszwaja/src/beam/sdks/java/extensions/avro/src/main/java/org/apache/beam/sdk/extensions/avro/schemas/utils/AvroUtils.java:538: error: [argument] incompatible argument for parameter namespace of AvroUtils.toAvroField.
      org.apache.avro.Schema.Field recordField = toAvroField(field, childNamespace);
                                                                    ^
  found   : @Initialized @Nullable String
  required: @Initialized @NonNull String
/usr/local/google/home/mszwaja/src/beam/sdks/java/extensions/avro/src/main/java/org/apache/beam/sdk/extensions/avro/schemas/utils/AvroUtils.java:541: error: [argument] incompatible argument for parameter name of Schema.createRecord.
    return org.apache.avro.Schema.createRecord(schemaName, "", schemaNamespace, false, fields);
                                               ^
  found   : @Initialized @Nullable String
  required: @Initialized @NonNull String
  ```

Reply:
I don't understand what this example has to do with isNullOrEmpty (which returns a boolean)?

Reply:
this is the error that appears when I use Strings.isNullOrEmpty

Reply:
understood